More documentation fixes/rewrites (#201)
dappnode/DAppNodeDocs: docs/user.md

slug: "/"
---

Welcome to the Dappnode user guide.
===
---
If you have just installed Dappnode on your own hardware or purchased a Dappnode from our [store](https://shop.dappnode.io/), the [Quick Start section](/user/quick-start/first-steps) will help you through the first steps with your Dappnode and get you on your way to exploring the Web3 ecosystem on your own terms.

> If you want to run your own validator, access decentralised apps or learn how to do cool stuff on your Dappnode visit the [Guides section](/user/guides/).

Reviewer: ```suggestion
If you want to run your own validator, access decentralised apps, or learn how to do other cool stuff on your Dappnode visit the [Guides section](/user/guides/).
```

If you have problems with your Dappnode or want to get in touch with us, please take a look at the [FAQs](/user/faq/general).

> The [product manual](/user/product-manual/dashboard) aims to give you an overview of the whole Dappnode interface, accompanied by a functional explanation of that part of the UI.

Reviewer: ```suggestion
The [product manual](/user/product-manual/dashboard) aims to give you an overview of the entire Dappnode interface, accompanied by a functional explanation of the AdminUI, the primary dashboard for managing your Dappnode.
```

> Welcome to the DAppNode user guide.

Reviewer: ```suggestion
Welcome to the Dappnode User Guide!
```

Author: That's already resolved. Make sure your commit filter is set to newest.

Reviewer: Ughh. Idk how that happened. I clicked to open what was the most recent requested review when I was working on it last night. I Thought it was redundant and had been done but just kept going.

===
---

> If you have just installed DAppNode on your own hardware or purchased a DAppNode from our [store](https://shop.dappnode.io/), the [Quick Start section](/user/quick-start/first-steps) will help you through the first steps with your DAppNode.

Reviewer: ```suggestion
If you have just installed Dappnode on your own hardware or purchased a Dappnode from our [store](https://shop.dappnode.io/), the [Quick Start section](/user/quick-start/first-steps) will help you through the first steps with your Dappnode, and get you on your way to exploring the Web3 ecosystem on your own terms
```

If you want to run your own validator, access decentralised apps or learn how to do cool stuff on your DAppNode visit the [Guides section](/user/guides/).

> If you have problems with your DAppNode or want to get in touch with us, please take a look at the [FAQs](/user/faq/general).

Reviewer: ```suggestion
If you have problems with your Dappnode or want to get in touch with us, please take a look at the [FAQs](/user/faq/general).
```

Author: That's already resolved. Make sure your commit filter is set to newest.
